index.html

```diff
@@ -59,7 +59,7 @@
                 document.write(script.outerHTML);
             })();
         </script>
-    </body>
```

Comment:
Why are we removing the closing `</body>` tag? Is there a problem I'm not seeing?